test/compilable/cppmangle.d

```diff
@@ -450,3 +450,20 @@ version (Posix) // all non-Windows machines
     static assert(test37!int.mangleof == "_ZN5SPACE6test37IiEEiv");
 }
 
+/**************************************/
+// 15388
+
+extern (C++) void test15388(typeof(null));
+
+version (linux)
```

Comment:
I recommend `Posix` if the mangling is the same.

Reply:
I used this because it appeared further above in the same file.

Reply:
Ok.